client/src/components/MovieOverview/MovieOverview.backup

```diff
@@ -0,0 +1,43 @@
+ useEffect(() => {
+    async function fetchOwnData() {
+      try {
+        const response = await axios.get('/api/movies/all')
+        console.log(response)
+        const dataOwnApi = response.data
+
+        setMovies(
+          dataOwnApi.map(item => {
+            return {
+              id: item._id,
+              imgUrl:
+                'https://image.tmdb.org/t/p/w500/a26cQPRhJPX6GbWfQbvZdrrp9j9.jpg',
+              title: item._id,
+              year: '1999',
+              genre: 'Drama',
+            }
+          })
+        )
+
+        for (const el of dataOwnApi) {
+          const resTmdb = await axios.get('/api/movies/all')
+
+          setMovies(
+            dataOwnApi.map(ele => {
+              return {
+                id: ele._id,
+                imgUrl:
+                  'https://image.tmdb.org/t/p/w500/a26cQPRhJPX6GbWfQbvZdrrp9j9.jpg',
+                title: el._id === ele._id ? 'Prcoessed' : ele._id,
+                year: '1999',
+                genre: 'Drama',
+              }
+            })
+          )
+        }
+      } catch (error) {
+        console.error(error)
+      }
+    }
+
```

Comment:
Curious: did it work?

Reply:
It worked so far! The tmdb connection is missing. I want to try it out in a later point, because it looks really good!

I had to move the tmdb api call to the backend, because it needs the api key. Then I went with the other solution which I understood better :D